### ISSUE TITLE
When will we see a new version (i.e. v0.12.1)?

### ISSUE THREAD

**Issue by bachhuberdesign** [joltup/rn-fetch-blob#582]
Hi, first off I wanted to say thank you for your hard work maintaining this library; it is a staple of many production apps (including my own). 

My question is -- when will we see a new version released with the fixes that have been merged onto master? In my opinion, the RNFetchBlobReq.java changes are important enough to warrant a bugfix release, as the issues it fixes (i.e. crashing) are pretty huge.

**Comment by oleksandr-dziuban** [comment]
Yes, please, @Traviskn Could you please release the new npm version with this fix for Android? It's very important

**Comment by zhuanghongji** [comment]
yea, an urgent release will help someone who using the lib (v0.12.0) on production now to fix the crash, and help someone who will use the lib tomorrow.

**Comment by rishabh876** [comment]
Is anyone using it this way in production? I don't think new version is coming. 
`"rn-fetch-blob": "git+https://github.com/joltup/rn-fetch-blob.git"`

**Comment by ThaJay** [comment]
> Is anyone using it this way in production? I don't think new version is coming. `"rn-fetch-blob": "git+https://github.com/joltup/rn-fetch-blob.git"`

I'll give it a try but a release would be nice.

Is there another package that could be a drop in replacement?
